OKR.md

```diff
@@ -2,6 +2,27 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2018 Q4
+
+**Debuggability and Stability**
+- error handling is done correctly and error codes are part of the API - @jacobheun
+- libp2p has a network visualization tool - @jacobheun
+- js-libp2p has a daemon with a RPC API and a client library - @jacobheun
+- interop tests for go-libp2p and js-libp2p exist for core functionality - @vasco-santos
+- a testbed exists and can test 50,000 libp2p nodes (browser and node) - @jacobheun
+- the testbed can run large scale tests for the DHT and Pubsub - @vasco-santos
```

Comment:
L12 and L13 seem very similar to me and they are a HUGE undertaking, perhaps the most important one (right after finalizing IPNS and DHT).

I recommend focusing hard on those 2 Objectives: a) large scale testing and b) finalize Interoperability through feature parity (word them as you see fit)
-

Reply:
We separated `L12` and `L13` into two different OKRs because we thought that it would be good to get both of us involved more heavily in this. And I would have the ones related to my on going work on the dht + pubsub. However, if you prefer to have a single one of us focusing on it, we can rearrange the assignments.

I completelly agree that those should be the most important objectives of this quarter, and we definetelly have to get them by the end of Q4.

Reply:
I think it would be best to keep L12 and L13 separate. As they are massive efforts we're really going to need to pair on this, especially later in the quarter as dependencies are finished. My thoughts on the separation are that L12 is focused on base functionality and support of growth, whereas L13 is adding on features to the tests. Getting the core functionality done in L12 opens up L13 to start. L12 can then continue, with a focus on scaling to 50,000 nodes.

It would be great if we could take some time at team week to talk through this in more depth with the larger libp2p team and infra, to discuss what needs to happen for all of this to be completed by end of year. I'd love to have some clear milestones for the various teams before team week is over, to better set ourselves up to hit a testbed of that scale.